src/hera/cron_workflow.py

```diff
@@ -52,6 +54,7 @@ def __init__(
         parallelism: int = 50,
         service_account_name: Optional[str] = None,
         labels: Optional[Dict[str, str]] = None,
+        namespace: Optional[str] = 'default',
```

Comment:
I think `Optional` represents a `Union[{chosen_type}, None]` so perhaps this can be:
```suggestion
        namespace: str = 'default',
```